src/components/Form.tsx

```diff
@@ -91,13 +91,18 @@ export class UnwrappedForm extends Component<IFormWrappedProps> {
   }
 
   public render() {
-    const { showControls, title } = this.props,
+    const { showControls, title, layout, colon } = this.props,
       formModel = this.formManager.formModel,
       filteredFieldSets = filterFieldSets(this.fieldSets, { model: formModel }),
       className = cx(CLASS_NAME, this.props.className);
 
     return (
-      <Antd.Form layout="vertical" onSubmit={this.formManager.onSave} className={className}>
+      <Antd.Form
+        layout={layout || 'horizontal'}
```

Comment:
I think this could be fixed by setting a default in `formPropsDefaults`, or since it's not a formProp, we could add a 

```ts
export const sharedComponentPropsDefaults = {
  layout: 'horizontal',
  colon: false,
};
```

and add it to these components like formPropsDefaults is today.

Reply:
This sounds good, the only thing right now is that Card components default to vertical orientation and Form components default to horizontal (because of and design). Defaulting both to the same would in one way or another change some components on the app when this gets released. Wondering your thoughts on this?

Reply:
Actually, that shouldn't be a problem, it looks like we were defaulting to vertical previously for our forms. Will just make "vertical" the default then and that should work for both!

Reply:
Perfect! Although, before, why were't cards getting your `sharedComponentPropsDefaults` default?

Reply:
OH, because you are't using `sharedComponentPropsDefaults` in your display components because you are using `format` as a single prop and not `colon` and `layout` like you do in the other components. Why is that?